services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusController.kt

```diff
@@ -474,10 +526,10 @@ The id needs to reference an excavation notification.
         return HankeError.HAI2002
     }
 
-    @ExceptionHandler(OperationalConditionDateException::class)
+    @ExceptionHandler(CompletionDateException::class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     @Hidden
-    fun operationalConditionDateException(ex: OperationalConditionDateException): HankeError {
+    fun operationalConditionDateException(ex: CompletionDateException): HankeError {
```

Comment:
Maybe the function name needs to be changed as well? E.g. `fun completionDateException(..)`

Reply:
Thank you, I renamed the function.